src/core/platforms/platformutilities.cpp

```diff
@@ -107,7 +107,8 @@ QString PlatformUtilities::systemSharedDataLocation() const
   const static QString sharePath = QDir( QFileInfo( !QCoreApplication::applicationFilePath().isEmpty() ? QCoreApplication::applicationFilePath() : QCoreApplication::arguments().value( 0 ) ).canonicalPath()
                                          + QLatin1String( "/../share" ) )
                                      .absolutePath();
-  return sharePath;
+  const static QString environmentSharePath = QDir( QString( qgetenv( "QFIELD_SYSTEM_SHARED_DATA_PATH" ) ) ).absolutePath();
```

Comment:
Can you describe the order of precedence / influence of env/cmake etc. variables in the docstring and/or header inside this method?

Reply:
@m-kuhn , good point, done.